Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PipelineBrush/PipelineBrushPage.xaml

```diff
@@ -21,8 +21,10 @@
             <Border>
                 <Border.Background>
                     <media:PipelineBrush>
```

Comment:
Have a crazy idea... what if the inputs were all `MarkupExtension` compatible? And the `Effects` was made the `Content` Property?

E.g.

```xml
  <media:PipelineBrush Input="{media:AcrylicEffect}">
    <effects:BlendEffect Input="{media:BackdropEffect Source=Backdrop}">
      <effects:HueRotationEffect/>  
    </effects:BlendEffect>
    <effects:BlurEffect/>
    <effects:ExposureEffect/>
    <effects:GrayscaleEffect/>
    <effects:HueRotationEffect/>
  </media:PipelineBrush>
```

Reply:
All done now! 🎉

I've also completely removed the `IPipelineInput` interface now since I moved all the sources to a dedicated folder with all extension methods.
Much easier to use now, awesome idea!